browser/templates/browser/add_person.html

```diff
@@ -0,0 +1,135 @@
+{% extends "browser/change_base.html" %}
+{% load addcss %}
+
+{% block title %}Create Person{% endblock %}
+{% block main %}
+
+<div class="alert alert-mbl">
+    <div>
+        <span class="h1">Add  Person</span>
+
+    </div>
+    {% if person.authority %}
+    <div class="text-warning">
+        <span class="glyphicon glyphicon-barcode"></span>
+        {{ person.authority.conceptpower_uri|urlize }}
+    </div>
+    {% endif %}
+    {% if person.validated %}
+    <div>
+        <span class="glyphicon glyphicon-ok" style="color:green;"></span>&nbsp;
+        Validated by {{ person.validated_by }} on {{ person.validated_on }}.
+    </div>
+    {% endif %}
+</div>
```

Comment:
since this is a form to create a new person, the above two blocks are kind of useless, aren't they?

Reply:
Yes, they do not serve any purpose. Now, I am using change_person.html and it does not have these two blocks.